scripts/make_orog.py

```diff
@@ -0,0 +1,132 @@
+"""
+The run script for making the orography files for the experiment.
```

Comment:
Oh yeah, great point!

Reply:
I pushed some hefty changes. Let me know what you think.

Reply:
I think I will leave it here. It may be a step too far for users, especially NCO. It's pretty standard for systems to use explicit suffixes, even when making the script executable.